dali/operators/sequence/optical_flow/optical_flow.h

```diff
@@ -105,12 +114,19 @@ class OpticalFlow : public Operator<Backend> {
   }
 
   optical_flow::VectorGridSize ConvertGridSize(int grid_size) {
-    if (grid_size < 4) {
-      return optical_flow::VectorGridSize::UNDEF;
-    } else if (grid_size == 4) {
-      return optical_flow::VectorGridSize::SIZE_4;
+    switch (grid_size) {
+      case 1:
+        return optical_flow::VectorGridSize::SIZE_1;
+      case 2:
+        return optical_flow::VectorGridSize::SIZE_2;
+      case 4:
+        return optical_flow::VectorGridSize::SIZE_4;
+      case 8:
+        return optical_flow::VectorGridSize::SIZE_8;
+      default:
+        return optical_flow::VectorGridSize::UNDEF;
+        break;
     }
-    return optical_flow::VectorGridSize::MAX;
   }
```

Comment:
Ekhm.... no. Let's get rid of this magic enum.

Reply:
Done